src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -244,7 +244,15 @@ foam.CLASS({
         .add(this.slot(function(mode, prop, prop$label) {
 
           var errorSlot = prop.validateObj && prop.validationTextVisible ?
-            this.data.slot(prop.validateObj) :
+            this.data.slot([[prop.name], function() {
+              var self = this;
+              var ax = self.cls_.getAxiomByName(prop.name);
```

Comment:
What is the difference between 'ax' and 'prop'?

Reply:
they are the same thing, i was doing something else when prop didn't work.
updated, thanks

Reply:
Much better. Let's wait until after UX meeting before merging.